ksqldb-cli/src/main/java/io/confluent/ksql/cli/console/Console.java

```diff
@@ -614,6 +616,33 @@ private void printSourceDescription(final SourceDescription source) {
         "Statistics of the local KSQL server interaction with the Kafka topic "
             + source.getTopic()
     ));
+    for (QueryOffsetSummary queryOffsetSummary : source.getQueryOffsetSummaries()) {
```

Comment:
Would be good to have some kind of heading here, e.g. 

```suggestion
    writer().println("Consumers:");
    writer().println("");
    
    for (QueryOffsetSummary queryOffsetSummary : source.getQueryOffsetSummaries()) {
```